frappe/public/js/frappe/model/meta.js

```diff
@@ -208,8 +208,8 @@ $.extend(frappe.meta, {
 		});
 
 		if(default_print_format && default_print_format != "Standard") {
-			var index = print_format_list.indexOf(default_print_format) - 1;
-			print_format_list.sort().splice(index, 1);
+			print_format_list.pop(default_print_format);
```

Comment:
`pop` does not take any argument, it always removes last element.

And this solution does not fix the issue.